solc/CommandLineParser.cpp

```diff
@@ -578,7 +579,11 @@ General Information)").c_str(),
 		)
 		(
 			g_strExperimentalViaIR.c_str(),
-			"Turn on experimental compilation mode via the IR (EXPERIMENTAL)."
+			"Deprecated synonym of --via-ir."
+		)
+		(
+			g_strViaIR.c_str(),
+			"Turn on experimental compilation mode via the IR."
```

Comment:
```suggestion
			"Turn on compilation mode via the IR."
```

Reply:
It was actually in there twice, yeah :-).